core-services/prow/02_config/_config.yaml

```diff
@@ -1105,6 +1105,7 @@ tide:
     - do-not-merge/invalid-owners-file
     - bugzilla/invalid-bug
     repos:
+    - openshift/build-machinery-go
```

Comment:
Should this `openshift` repo really be outside of the BZ machinery?

Reply:
yes, library-go has exception, the split inherits it.

Reply:
see https://github.com/openshift/release/pull/7013